### PR TITLE
Webdriver

### PR DESCRIPTION
**Pull request by stasya72008**

### CODE REVIEW

File: script.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+from selenium import webdriver
+
+from constants import *
+from database import *
```
Comment (denyskulykov) [inline script.py]: Please, don't use  import All _*_

File: script.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+all_links_list = []
+
+create_bd()
+driver.get(URL_PATH)
+
+pages = driver.find_elements_by_css_selector(PAGE_LINK_CSS)
+pages_number = int(pages[-1].text) if pages else 1
+
+# Todo: add try-except blocks
+for i in range(1, pages_number + 1):
```
Comment (denyskulykov) [inline script.py:27]: There is the opening of first page twice

File: script.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    driver.get(URL_PATH + '&page={}'.format(i))
+    elem_list = driver.find_elements_by_css_selector(ITEMS_LINKS_CSS)
+    links_list = list()
+
+    for element in elem_list:
+        # Todo (specially for prune) - add check for olx
```
Comment (denyskulykov) [inline script.py:33]: ppppp

Comment (stasya72008) [reply script.py:33]: XD. Is something wrong?)

**Pull request merged**